src/plugins/dashboard/public/dashboard_actions/replace_panel_flyout.tsx

```diff
@@ -92,7 +77,9 @@ export class ReplacePanelFlyout extends React.Component<Props> {
       />
     );
 
-    const panelToReplace = 'Replace panel ' + this.props.panelToRemove.getTitle() + ' with:';
+    const panelToReplace =
+      'Replace panel ' + this.props.api.panelTitle?.value ??
```

Comment:
Just noticed this string is not passing thru i18n.

Reply:
Good catch, fixed this!